Makefile.dep

```diff
@@ -555,3 +530,13 @@ ifneq (,$(filter random,$(USEMODULE)))
         USEMODULE += tinymt32
     endif
 endif
+
+# include package dependencies
+-include $(USEPKG:%=$(RIOTPKG)/%/Makefile.dep)
```

Comment:
Arghs... Should have looked at this more careful. Now one NEEDS to mention a packet right? (except lwIP which is still above for some weird reason).